CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [#6109](https://github.com/rubocop-hq/rubocop/pull/6109): Add new `Bundler/GemComment` cop. ([@sunny][])
 * [#6148](https://github.com/rubocop-hq/rubocop/pull/6148): Add `IgnoredMethods` option to `Style/NumericPredicate` cop. ([@AlexWayfer][])
+* [#6174](https://github.com/rubocop-hq/rubocop/pull/6174): Add `--fail-level-output` to only output offenses at or above the fail level. ([@robotdana][])
```

Comment:
Updating description of CHANGELOG.md from `--fail-level-output` to `--display-only-fail-level-offenses` would be kind to users.

Reply:
Oops I missed this, thank you :)